eth/client.go

```diff
@@ -409,7 +412,7 @@ func (c *Client) WithdrawDeposit() (<-chan types.Receipt, <-chan error) {
 
 func (c *Client) Job(streamId string, transcodingOptions string, maxPricePerSegment *big.Int) (<-chan types.Receipt, <-chan error) {
 	return c.WaitForReceipt(func() (*types.Transaction, error) {
-		if tx, err := c.jobsManagerSession.Job(streamId, transcodingOptions, maxPricePerSegment); err != nil {
+		if tx, err := c.jobsManagerSession.Job(streamId, transcodingOptions, maxPricePerSegment, big.NewInt(0)); err != nil {
```

Comment:
The endBlock parameter should be non-zero and set to some default value like currentBlock + N and N could be roundLength. We could also let the user specify a custom endBlock, but we should set a default at the very least